frontend/src/scenes/feature-flags/FeatureFlags.tsx

```diff
@@ -177,11 +182,54 @@ export function FeatureFlags(): JSX.Element {
                 nouns={['feature flag', 'feature flags']}
                 data-attr="feature-flag-table"
             />
+        </>
+    )
+}
+
+export function FeatureFlags(): JSX.Element {
+    const { featureFlags } = useValues(featureFlagLogic)
+    const showActivityLog = featureFlags[FEATURE_FLAGS.FEATURE_FLAGS_ACTIVITY_LOG]
+    return (
+        <div className="feature_flags">
+            <PageHeader
+                title="Feature Flags"
+                caption={
+                    showActivityLog
+                        ? ''
+                        : 'Feature Flags are a way of turning functionality in your app on or off, based on user properties.'
+                }
+                buttons={
+                    showActivityLog ? (
+                        <LinkButton
+                            type="primary"
+                            to={urls.featureFlag('new')}
+                            data-attr="new-feature-flag"
+                            icon={<PlusOutlined />}
+                        >
+                            New Feature Flag
+                        </LinkButton>
+                    ) : (
+                        <></>
+                    )
```

Comment:
nit `: null` is less code than the `React.createElement(React.fragment(...))` that happens with `<></>`

Reply:
Oh, you two always making me add nulls everywhere 🤣  #BillionDollarMistake :)

Reply:
Ah, you can't pass null here

![Screenshot 2022-03-21 at 12 24 18](https://user-images.githubusercontent.com/984817/159260377-f8ddadff-c68d-4db9-9783-0533f65d51cc.png)

This gets removed when we remove the feature flag

Reply:
You can pass `false` or `undefined` though... so plenty of negativity to go around 😅

Reply:
Will split the difference and pass false :)